src/MonitorProfiler/ProbeInstrumentation/ProbeInstrumentation.cpp

```diff
@@ -8,6 +8,12 @@ using namespace std;
 
 #define IfFailLogRet(EXPR) IfFailLogRet_(m_pLogger, EXPR)
 
+#ifndef DLLEXPORT
+#define DLLEXPORT
+#endif
+
+BlockingQueue<PROBE_WORKER_PAYLOAD> ProbeInstrumentation::s_probeManagementQueue;
```

Comment:
Rather than making this a static member of `ProbeInstrumentation`, make it a global static and rename the prefix to `g_`. This seems to only be accessible from this file; making this change alleviates the need to make this publicly available to anything that includes ProbeInstrumentation.h